policyengine_us/parameters/gov/states/ma/tax/income/credits/dependent/amount.yaml

```diff
@@ -1,12 +1,20 @@
 description: MA dependent credit amount.
 values:
   2021-01-01: 180
+  2023-01-01: 310
+  2024-01-01: 440
```

Comment:
delete for now and file an issue for 2024 child and dependent tax credit changes and include this reference:
https://www.mass.gov/news/governor-healey-signs-first-tax-cuts-in-more-than-20-years